README.md

```diff
@@ -24,45 +27,39 @@ This example introduces a contract that verifies a user's passport birthday and
    ```
 
 3. Configure environment variables:
-   - Copy `.env.example` to `.env`
-   - Fill in the required values in the `.env` file
+   - Copy `.env.example` to `.env` (or create a new `.env` file)
+   - Add the following required values:
+   ```env
+   # Private key for deployment (without 0x prefix)
+   CELO_ALFAJORES_KEY=your_private_key_here
+   
+   # Celoscan API key for contract verification (optional but recommended)
+   CELOSCAN_API_KEY=your_celoscan_api_key_here
+   ```
 
 4. Build the contracts:
    ```bash
    yarn run build
    ```
 
-5. Configure the passport environment in the `contracts/ignition/modules/SelfHappyBirthday.ts` file:
-   - For real passports (production environment):
-     Uncomment the line for production and comment the staging line:
+5. Configure the passport environment in `contracts/scripts/hardhat/deployHappyBirthday.ts`:
+   - For **mock passports** (testing/development):
      ```javascript
-     // For prod environment
-     const DEFAULT_IDENTITY_VERIFICATION_HUB = "0x77117D60eaB7C044e785D68edB6C7E0e134970E";
-     // For staging environment
-     // const DEFAULT_IDENTITY_VERIFICATION_HUB = "0x3e2487a250e2A7b56c7ef5307Fb591Cc8C83623D";
-     // For mainnet environment
-     const token = "0xcebA9300f2b948710d2653dD7B07f33A8B32118C";
-     // For staging environment
-     // const token = "0x2F25deB3848C207fc8E0c34035B3Ba7fC157602B";
+     devMode: true,
```

Comment:
This line should be in the deployHappyBirthday.ts file, and then have a comment line above to comment it out for prod

Reply:
In the file you are defining the verification first like this. Should be consistent in README and deployHappyBirthday file

  const devMode = true;
  const ofacEnabled = [true, true, true] as [boolean, boolean, boolean];